src/constants.ts

```diff
@@ -95,6 +95,7 @@ export const NUMBER_SOURCES_PER_LOG_LINE = 12;
 export const RFQ_FIRM_QUOTE_CACHE_EXPIRY = ONE_MINUTE_MS * 2;
 export const RFQ_ALLOWANCE_TARGET = '0xdef1c0ded9bec7f1a1670819833240f027b25eff';
 export const RFQM_MINIMUM_EXPIRY_DURATION_MS = ONE_MINUTE_MS * 3;
+export const RFQM_TX_GAS_ESTIMATE = 165e3;
```

Comment:
Thoughts on naming? Something like `RFQM_FEE_GAS_MULTIPLIER`

Reply:
RFQM_TX_GAS_ESTIMATE is fine I think. just a rough gas unit usage guess